doc/source/whatsnew/v0.21.0.txt

```diff
@@ -135,6 +135,7 @@ Removal of prior version deprecations/changes
 Performance Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+- Fixed performance of instantiating :class:`SparseDataFrame` (:issue:`16773`)
```

Comment:
Maybe I'm being pedantic, but you can't really "fix" performance, only improve it. 😄